### DISCUSSION TITLE
Option to stop current revision edits from updating drafts.

### DISCUSSION THREAD

**Discussion by JordanMed** [craftcms/cms#12025] [Ideas]
Currently my team uses drafts as a way to build new test/demo pages of existing pages. This way we can try out new changes and show the results too management with out editing pages that are live on the website. The problem is our current revisions still require changes and updates too content. Some of these drafts can be worked on for months before being approved/applied, meaning every-time we make an update to the current revision it also updates our draft pages with this new content which is unwanted. 

Maybe there is already a solution to this that I don't know about but this is something my team could use.

**Comment by brandonkelly** [comment]
Interesting. Do you think this should be a global config setting, or something that each individual draft should get to opt out of?

**Reply by JordanMed** [reply]
I think both of those options would be great! As I could see us using both across our multiple websites. I was thinking a global setting would be nice just because we have multiple people working on our sites at the same time, so trying to coordinate everyone is always hard. But giving the option on the entry its self would probably be the best option for flexibility.
